packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts

```diff
@@ -0,0 +1,47 @@
+// @group:suite
+// @retry=2
+
+describe('Go into new accounts', () => {
+        beforeEach(() => {
+             // Launches Trezor emulator
+            cy.task('startEmu', { wipe: true });
+            //Initializes Trezor (random seed)
```

Comment:
Let's always add a space after the `//`. So `// Initializes Trezor (random seed)`. Let's do this everywhere.

Reply:
534889de5464e01a3eb5970394719506dc079b1e done